server/build/docker-compose.common.yml

```diff
@@ -53,6 +53,7 @@ services:
     command: postgres -c 'config_file=/etc/postgresql/postgresql.conf'
     volumes:
      - "./docker/postgres.conf:/etc/postgresql/postgresql.conf"
+     - "./docker/postgres_node_database.sql:/docker-entrypoint-initdb.d/postgres_node_database.sql"
```

Comment:
I don't understand this path: `/docker-entrypoint-initdb.d/`. Is this something the postgres docker container automatically looks at and executes any scripts in this dir?

Reply:
Exactly, both sql and sh are run on init if the data directory is empty (here are the docs, info is in the [Initialization scripts](https://hub.docker.com/_/postgres) section)

Reply:
Great stuff. Thanks.

Reply:
Probably we could use this script to even create and initialize the actual `mattermost_test` DB. Seems like a useful thing. But out of scope for this PR.